recipe/meta.yaml

```
@@ -54,26 +54,23 @@ outputs:
 
   - name: psycopg
     build:
+      number: 1
       script: cd psycopg && {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
       noarch: python
-      skip: true  # [osx or build_platform != target_platform or py!=39]
+      skip: true  # [py!=39]
```

This needs to skip all but one OS as well
```suggestion
      skip: true  # [not linux64 or py!=39]
```
Here's a similar case:
https://github.com/conda-forge/airflow-feedstock/blob/main/recipe/meta.yaml#L211